File: lista2/Ex4.java
package lista2;

import java.util.Scanner;

public class Ex4 {
    public static void main(String[] args) {
        
        Scanner input = new Scanner(System.in);

        double notaGA;
        double notaGB;

        System.out.print("Digite sua nota no grau A: ");
        notaGA = input.nextDouble();

        System.out.print("Digite sua nota no grau B: ");
        notaGB = input.nextDouble();

        double notaFinal = (notaGA*3.33 + notaGB*6.67)/10;

        if(notaFinal >= 6) {
            System.out.println("Sua nota final no semestre é "+notaFinal+". Parabéns! Você passou.");
        }
        else {
            System.out.println("Sua nota no semestre é "+notaFinal+". Vai precisar fazer Grau C! :(");
        }
    }
}

File: lista2/Ex4.c
#include <stdio.h>

int main() {

    double notaGA;
    double notaGB;

    printf("Digite sua nota no grau A: ");
    scanf("%lf", &notaGA);

    printf("Digite sua nota no grau B: ");
    scanf("%lf", &notaGB);

    double notaFinal = (notaGA*3.33 + notaGB*6.67)/10;
    
    if(notaFinal >= 6) {
        printf("Sua nota final no semestre é %lf. Parabéns!\n", notaFinal);
    } else {
        printf("Sua nota final no semestre é %lf. Vai precisar fazer grau C! :(\n", notaFinal);
    }

    return 0;
}